buildenv/jenkins/JenkinsfileBase

```diff
@@ -364,6 +364,21 @@ def setup() {
 		} else if (params.CUSTOMIZED_SDK_URL) {
 			SDK_RESOURCE = "customized"
 			CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL}'"
+			if (params.ADDITIONAL_ARTIFACTS_REQUIRED == "RI_JDK") {
+				def server = Artifactory.server params.ARTIFACTORY_SERVER
+				def artifactoryUrl = server.getUrl()
+				def repoForRi = ''
+				def riURL = ''
+				if (params.ARTIFACTORY_REPO.contains(',')) {
+					String[] repos = params.ARTIFACTORY_REPO.split(",")
+					// Assumption: ARTIFACTORY_REPO=X,Y (X=to upload results, Y=to download ri)
+					repoForRi = repos[1].trim()
+					if (!env.SPEC.startsWith('aix') && !env.SPEC.startsWith('zos')) {
+						riURL = "${artifactoryUrl}/${repoForRi}/Latest/${PLATFORM}/${JDK_VERSION}"
+					}
+				} 
+				CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL} ${riURL}'"
```

Comment:
I thought the plan was to allow users to provide the RI JDK URL in CUSTOMIZED_SDK_URL (according to https://github.com/adoptium/aqa-tests/issues/4350#issuecomment-1453817503). Did we change the design? 
This code basically sets the RI JDK URL automatically for users and leaves them with limited options.

A couple of assumptions in this code that may not be true:
- users only use artifactory server to store RI JDK
- users use the same artiftactory server `params.ARTIFACTORY_SERVER` for storing test results and RI JDK
- the second position in params.ARTIFACTORY_REPO has to be for RI JDK. If not provided, riURL will be empty

Reply:
Hi Lan, Yes, the design did change a little bit in that we are now dynamically constructing the RI JDK path based on the assumptions you pointed out above. We are trying to solve the multijvm case here - where these assumptions hold true. Pplan is to add the other scenarios (as outlined in https://github.com/adoptium/aqa-tests/issues/4350#issuecomment-1453817503) as enhancements in followup PRs.